src/types.jl

```diff
@@ -71,23 +73,24 @@ end
 Mixture
 =#
 
-struct BrusilovskyEoSMixture{T} <: AbstractEoSMixture
-    components::Vector{BrusilovskyEoSComponent{T}}
+struct BrusilovskyEoSMixture{T,VC} <: AbstractEoSMixture
+    components::VC
 
     eij::Matrix{T} # constant  thermal binary interaction coefficient
     gij::Matrix{T} # linear    thermal binary interaction coefficient
     hij::Matrix{T} # quadratic thermal binary interaction coefficient
+end
 
-    function BrusilovskyEoSMixture(
-        ;
-        components::AbstractVector{BrusilovskyEoSComponent{T}},
-        constant::AbstractMatrix,
-        linear::AbstractMatrix,
-        quadratic::AbstractMatrix,
-        kw...
-    ) where {T}
-        new{T}(components, constant, linear, quadratic)
-    end
+const BrusilovskyEoSMixture{T} = BrusilovskyEoSMixture{T,VC} where {VC}
```

Comment:
Type alias for legacy code.

Reply:
VSCode complains on invalid redefinition of a constant.
In fact, `BrusilovskyEoSMixture{T}` is already that by default.

Reply:
Perhaps, there was a mess with changing of base branch of the pull request. The actual code is

```julia
struct BrusilovskyEoSMixture{T,VC} <: AbstractEoSMixture
    components::VC

    eij::Matrix{T} # constant  thermal binary interaction coefficient
    gij::Matrix{T} # linear    thermal binary interaction coefficient
    hij::Matrix{T} # quadratic thermal binary interaction coefficient
end

const BrusilovskyEoSMixture{T} = BrusilovskyEoSMixture{T,VC} where {VC}

function BrusilovskyEoSMixture(;
    components::AbstractVector{<:BrusilovskyEoSComponent},
    constant::AbstractMatrix,
    linear::AbstractMatrix,
    quadratic::AbstractMatrix,
    kw...
)
    return BrusilovskyEoSMixture(StructArray(components), constant, linear, quadratic)
end
```

So, `BrusilovskyEoSMixture{T,VC}` is two-parameter type.

Reply:
I removed the type alias in 144ae05754962940ce5cd67f18f691051de30b99.